test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -163,6 +167,89 @@ var _ = Describe("[performance] Checking IRQBalance settings", func() {
 
 			Expect(postRestartBannedCPUs.ToSlice()).To(Equal(postCreateBannedCPUs.ToSlice()), "banned CPUs changed post tuned restart on node %q", postRestartBannedCPUs.ToSlice(), targetNode.Name)
 		})
+
+		It("Should store empty cpu mask in the backup file", func() {
+			// crio stores the irqbalance CPU ban list in the backup file once, at startup, if the file doesn't exist.
+			// This _likely_ means the first time the provisioned node boots, and in this case is _likely_ the node
```

Comment:
The comment is not so clear
What  \_\_likely\_\_ convention means? I know this term from the C programming world but I don't understand how it fits in here

Reply:
this is just the word "likely" (in plain english meaning) emphasized in the text :) Is intended to be read as underlined text for emphasis